jvm-libs/linea/metrics/micrometer/build.gradle

```diff
@@ -7,4 +7,5 @@ description="Micrometer metrics implementation for Linea"
 dependencies {
   api "io.vertx:vertx-micrometer-metrics"
```

Comment:
Can you please get rid of it in favor of
```
api "io.micrometer:micrometer-registry-prometheus:${libs.versions.micrometer.get()}"
```
 like I did in my [change](https://github.com/Consensys/linea-monorepo/pull/1107/files#diff-f6a4e0528e5f7de546e6ad297253a5b2bbb23b812827f06cd211aecd9976428eR8)?